### ISSUE TITLE
Frontend: Create dashboard

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#31]
## User story
1. As a {HR-employee}
2. I want / need {to have a dashboard which is the page that I see after login on DIdentity frontend}
3. So that {I have an overview over the functionalities and that I can choose what I want to do next}

## Acceptance criteria
* Design dashboard according to the mockup provided
* Create dynamic behaviour: When clicking on the different fields, the respective page opens (e.g. overview page, "create new identity"-page)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Finished in sprint 4:  b0d9955e98f9a9708c55925ad923adabc86005db
